content/en/blog/2023/euwg-future-of-observability-panel/index.md

```diff
@@ -0,0 +1,99 @@
+its standardization of telemetry data and its vendor-neutrality.
+
+### 2- OpenTelemetry as a Game Changer
+
+OpenTelemetry emerged as a game-changing solution for enabling Observability. It
+provided a standardized approach to instrumenting applications. The adoption of
+OpenTelemetry allowed for seamless integration across various systems and
+languages. The community-driven and open source nature of OpenTelemetry appealed
+to many organizations, making Observability more accessible and
+developer-friendly.
+
+### 3- The OpenTelemetry Collector
+
+The panelists talked about the [OpenTelemetry Collector](/docs/collector/), and
+how they were able to use it to replace vendor-specific agents. Some
+organizations built their own Collector distributions to meet their own
+requirements using the
+[Collector Builder tool](/docs/collector/custom-collector/) tool. In doing so,
+they were able to include only the components that were applicable to their use
+case, including using custom processors.
+
+### 4- Surprising Aspects of OpenTelemetry Adoption
+
+Challenges included convincing teams to adopt OpenTelemetry, especially when
+components like Logs were not yet stable. The evolving nature of OpenTelemetry
+made sometimes challenging to keep up with new features and updates while
+ensuring that teams did not lag behind.
+
+### 5- The People Problem of Observability
+
+Observability, like so many issues in the tech industry is more of a people
+problem than a technical problem. Successful OpenTelemetry adoption and
+implementation required addressing cultural and organizational challenges,
+including leadership buy-in and developer acceptance.
+
+## Video recording
+
+You can catch the recording of this discussion below:
+
+{{<youtube zSeKL2-_sVg>}}
+
+Don't forget to subscribe to our
+[YouTube channel](https://youtube.com/@otel-official) for more great
+OpenTelemetry content!
+
+## Join us!
+
+If you have a story to share about how you use OpenTelemetry at your
+organization, we’d love to hear from you! Ways to share:
+
+- Join the [#otel-endusers channel](/community/end-user/slack-channel/) on the
+  [CNCF Community Slack](https://communityinviter.com/apps/cloud-native/cncf)
+- Join our monthly
+  [End-User Discussion Group calls](/community/end-user/discussion-group/)
+- Join our [OTel in Practice](/community/end-user/otel-in-practice/) sessions
+- Share your stories on the
+  [OpenTelemetry blog](https://github.com/open-telemetry/opentelemetry.io/blob/954103a7444d691db3967121f0f1cb194af1dccb/README.md#submitting-a-blog-post)
+- Contact us on the
+  [CNCF Community Slack](https://communityinviter.com/apps/cloud-native/cncf)
+  for any other types of sessions you'd like to see!
+
+Be sure to follow OpenTelemetry on
+[Mastodon](https://fosstodon.org/@opentelemetry) and
+[X](https://x.com/opentelemetry), and
+[LinkedIn](https://www.linkedin.com/company/opentelemetry/), and share your
```

Comment:
```suggestion
[Mastodon](https://fosstodon.org/@opentelemetry) and
[LinkedIn](https://www.linkedin.com/company/opentelemetry/), and share your
```

The twitter feed hasn't been updated since September last year so I think we can skip it here ...